[CUBRIDMAN-182] support with deduplicate clause (#448)
CUBRID/cubrid-manual: en/sql/schema/table_stmt.rst

*   *table_comment_string*: specifies a table's comment
*   *column_comment_string*: specifies a column's comment.
*   *index_comment_string*: specifies an index's comment.

> *   *deduplicate_level*: Specifies the deduplicate level (0 to 14). For details, see :ref:`deduplicate_overview`.

Reviewer: ```suggestion
*   *deduplicate_level*: specifies the deduplicate level (0 to 14). For details, see :ref:`deduplicate_overview`.
```

.. note::

    *   **DBA** and **DBA** members can create tables in different schemas. If a user is neither **DBA** nor **DBA** member, tables can only be created in the schema of that user.

.. note::

>     *deduplicate_level*\ is an integer from 0 to 14. 0 means an index with the same configuration as CUBRID 11.2 or earlier without the **DEDUPLICATE** option.

Reviewer 2: ```suggestion
    *deduplicate_level*\ is an integer from 0 to 14. When this parameter is set to 0, it indicates an index configuration where data deduplication within indexes is not performed. This configuration is the same as that in CUBRID 11.2 or an earlier version.
```

Author: Thank you.

>     *deduplicate_level*\ is an integer from 0 to 14. 0 means an index with the same configuration as CUBRID 11.2 or earlier without the **DEDUPLICATE** option.

Reviewer: ```suggestion
    *deduplicate_level*\ is an integer from 0 to 14. 0 means an index with the same configuration as CUBRID 11.2 or earlier which doesn't support the **DEDUPLICATE** option.
```

Author: @mhoh3963 
This sentence will be corrected to accommodate hgryoo's comment above.

    *   **SET NULL**: When a specific record is being deleted or updated, the column value of the foreign key is updated to **NULL**.
    *   **NO ACTION**: Its behavior is the same as that of the **RESTRICT** option.

> *   *deduplicate_level*: Specifies the deduplicate level (0 to 14). For details, see :ref:`deduplicate_overview`.

Reviewer: ```suggestion
*   *deduplicate_level*: specifies the deduplicate level (0 to 14). For details, see :ref:`deduplicate_overview`.
```

                        ON DELETE <referential_action> 

                        <referential_action> ::= CASCADE | RESTRICT | NO ACTION | SET NULL
                        
                <index_with_option> ::= {DEDUPLICATE ‘=‘ deduplicate_level}                        

> *   *schema_name*: Specifies the schema name. If omitted, the schema name of the current session is used.

Reviewer: ```suggestion
*   *schema_name*: specifies the schema name. If omitted, the schema name of the current session is used.
```
